support-figma/extended-layout-plugin/src/ui.html

```diff
@@ -385,6 +386,18 @@
 </div>
 <!-- END OF LOCALIZATION PLUGIN UI -->
 
+<!-- BEGIN OF IMAGE RES PLUGIN UI -->
+<div id="imageExport" class="page page-padding-large">
+  <div style="border-bottom: solid #000000; font-weight: bold;">Images:</div>
+  <div style="font-size: 14; margin-top: 8px;">* Jpg or png images are downloaded in png format, gif images are downloaded in gif format but DesignCompose doesn't support gif.</div>
```

Comment:
Not sure what the '*' is for at the beginning, perhaps remove it?

Reply:
Those are instructions or explanations of the plugin. I changed it to <li></li>.